docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -308,7 +308,8 @@ blockquote:nth-child(n) {
 }
 
 a[href*="//"].hs-external-link,
-.hs-external-link a[href*="//"]
+.hs-external-link a[href*="//"],
+.ck-content a.hs-external-link
```

Comment:
Sorry, I missed this one on the previous release, but this is also specific to the editor and shouldn't be in the FE styles. Can you please move this to `docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss`?